capabilities/blob/replica/allocate.go

```diff
@@ -0,0 +1,57 @@
+package replica
+
+import (
+	"github.com/ipld/go-ipld-prime/datamodel"
+	"github.com/storacha/go-ucanto/core/ipld"
+	"github.com/storacha/go-ucanto/core/schema"
+	"github.com/storacha/go-ucanto/did"
+	"github.com/storacha/go-ucanto/ucan"
+	"github.com/storacha/go-ucanto/validator"
+
+	"github.com/storacha/go-libstoracha/capabilities/space/blob"
```

Comment:
feels a bit gross to import this way, and I'm not really sure it even matters, but since this is referencing a "space" blob I've done so. Thinking it might be best to move `Blob` to `capabilities/types` since it's used everywhere. Will do so in a separate commit so reverting is easy.